### DISCUSSION TITLE
Is there a simple way to count how many entries a timeserie contains?

### DISCUSSION THREAD

**Discussion by ynnova** [ravendb/ravendb#18145] [Q&A]
The context of the questions is around the concept of pagination.
I would like to paginate a timeserie containing a lot of entries at (lets say) 100 items per page.
In order to arrange the pagination user interface I need the number of entries that a timeseries contains.
A sort of session.Statistics(...).

I cannot find anything in the documentation.
I imagine there is a simple way but probably i cannot see in the right place.

Any help?

L.

**Comment by karmeli87** [comment]
Hi,

It seems we miss this functionality in our documentation.

In the meanwhile here is what you need:

```
var stats = await store.Operations.SendAsync(new GetTimeSeriesStatisticsOperation(documentId));
```

This will fetch statistics for every time-series associated with the given document.
